lib/gc/Analysis/MatmulConfigAnalysis.cpp

```diff
@@ -393,10 +400,10 @@ bool readConfigFromAttrs(MatmulConfig &config, ArrayRef<NamedAttribute> attrs) {
       cfgItemCnt++;
     }
   }
-  if (validateConfig(config)) {
+  if (validateConfig(config, shape)) {
     return cfgItemCnt == 9;
   } else {
-    LLVM_DEBUG(llvm::dbgs() << "The predefined config is invalid\n");
+    assert(validateConfig(config, shape) && "config is invalid");
```

Comment:
@zhczhong Replaced the original debug info with assertion to avoid invalid config being replaced with a valid one, which potentially will lead to error in tuning result.